api/agent/agent.go

```diff
@@ -1235,6 +1237,11 @@ func newHotContainer(ctx context.Context, evictor Evictor, call *call, cfg *Conf
 		},
 	}
 
+	auther := call.dockerAuth
```

Comment:
This is to make the tests pass, since TestDockerAuthExtn does not have a slot queue/mgr.